SEGSupport/CMakeLists.txt

```diff
@@ -1,29 +1,42 @@
 #-----------------------------------------------------------------------------
-set(MODULE_NAME SEG2NRRD)
+# Not used
+set(MODULE_NAME SEGSupport)
 
 #-----------------------------------------------------------------------------
 set(MODULE_INCLUDE_DIRECTORIES
   )
 
-set(MODULE_SRCS
-  SEG2NRRD.cxx
-  framesorter.h
-  SegmentAttributes.h
-  )
-
+#-----------------------------------------------------------------------------
 set(MODULE_TARGET_LIBRARIES
+  ${DCMTK_LIBRARIES}
   ${ITK_LIBRARIES}
-  )
+  xml2 z
+)
 
 #-----------------------------------------------------------------------------
 SEMMacroBuildCLI(
-  NAME ${MODULE_NAME}
-  TARGET_LIBRARIES ${MODULE_TARGET_LIBRARIES}
+  NAME EncodeSEG
   INCLUDE_DIRECTORIES ${MODULE_INCLUDE_DIRECTORIES}
-  ADDITIONAL_SRCS ${MODULE_SRCS}
-  )
+  ADDITIONAL_SRCS EncodeSEG.cxx
+  TARGET_LIBRARIES ${MODULE_TARGET_LIBRARIES}
+  RUNTIME_OUTPUT_DIRECTORY        "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}"
+  LIBRARY_OUTPUT_DIRECTORY        "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}"
+  ARCHIVE_OUTPUT_DIRECTORY        "${CMAKE_ARCHIVE_OUTPUT_DIRECTORY}"
+  INSTALL_RUNTIME_DESTINATION ${CMAKE_BINARY_DIR}
+  INSTALL_LIBRARY_DESTINATION ${CMAKE_BINARY_DIR}
+  INSTALL_ARCHIVE_DESTINATION ${CMAKE_BINARY_DIR}
```

Comment:
@jcfr I think the issue is that not all developers know how this macro is actually working, and just follow examples from other places (at least that is what I do).

I checked another extension, and guess we need to update to follow that pattern instead: https://github.com/fedorov/ChangeTrackerPy/blob/master/Metrics/IntensityDifference/CMakeLists.txt#L13-L18